packages/frontend/package.json

```diff
@@ -66,9 +66,11 @@
     "prettier": "^2.6.2",
     "prettier-plugin-tailwindcss": "^0.2.2",
     "sass": "^1.52.2",
-    "storybook-dark-mode": "^1.1.0",
+    "storybook": "^7.0.9",
+    "storybook-dark-mode": "^3.0.0",
     "storycap": "^3.1.9",
     "tailwindcss": "^3.1.0",
+    "ts-node": "^10.9.1",
```

Comment:
new dep?

Reply:
When I started using ts files for storybook (main and preview), build started to fail, adding this as a dev dependency fixed the issue